ispyb-ejb/src/main/java/ispyb/server/mx/services/utils/reader/AutoProcProgramaAttachmentFileReader.java

```diff
@@ -157,15 +269,18 @@
 		// o[0] is a boolean xscaleFile
 		// o[1] is a boolean truncateLog
 		// o[2] is List<AutoProcessingData>
+		// o[2] is mergedNoanomCorrectFile
```

Comment:
Minor: o[3] 
but I think that in fact we should remove all these comments, they are useless, what do you think ?